tests/test_skySources.py

```diff
@@ -24,6 +24,7 @@
 import logging
 
 import lsst.afw.image as afwImage
+import lsst.meas.extensions.piff.piffPsfDeterminer
```

Comment:
Why is this now needed?

Reply:
Because I tried to run the test file on its own (it's the slowest test now) and it didn't work. The test code was relying on other test code doing that import.

Reply:
Is that telling me there is a file in pipe_tasks itself that is not doing this import? I'm not really sure how the meas_extensions import stuff works.